docs/flatcar-linux/packet.md

```diff
@@ -307,20 +292,35 @@ Check the [variables.tf](https://github.com/kinvolk/lokomotive-kubernetes/blob/m
 
 #### DNS Zone
 
-Clusters create a DNS A record `${cluster_name}.${dns_zone}` to resolve a network load balancer backed by controller instances. This FQDN is used by workers and `kubectl` to access the apiserver(s). In this example, the cluster's apiserver would be accessible at `tempest.aws.example.com`.
+Clusters create few DNS A records to resolve to controller instances. For instance `${cluster_name}.${dns_zone}` is used by workers and `kubectl` to access the apiserver(s). In this example, the cluster's apiserver would be accessible at `tempest.myclusters.example.com`.
 
-You'll need a registered domain name or delegated subdomain on AWS Route53. You can set this up once and create many clusters with unique names.
+In order to create such DNS entries you'll need a registered domain/subdomain and to define a DNS module that takes as input the DNS entries required by the controller module.
 
 ```tf
-resource "aws_route53_zone" "zone-for-clusters" {
-  name = "aws.example.com."
+module "dns" {
+  source = "git::https://github.com/kinvolk/lokomotive-kubernetes//dns/providerx?ref=<hash>"
```

Comment:
Please write `<provider>` instead of `providerx`.

Reply:
Done.